### PR TITLE
Update VCPKG

### PR DESCRIPTION
**Pull request by JulianGro**
This PR updates VCPKG for Windows amd64, Linux amd64, Linux aarch64, and macOS amd64.

### CODE REVIEW

**Comment by AnimatedMannequin** [comment]
Here are some builds you may use for testing: 

**macOS-10.15, full**
 - https://public.overte.org/build/overte/PR162/Overte-PR162-bb58e71.dmg

**windows-2019, full**
 - https://public.overte.org/build/overte/PR162/Overte-PR162-bb58e71.exe

**Comment by AnimatedMannequin** [comment]
Here are some builds you may use for testing: 

**macOS-10.15, full**
 - https://public.overte.org/build/overte/PR162/Overte-PR162-4600443.dmg

**windows-2019, full**
 - https://public.overte.org/build/overte/PR162/Overte-PR162-4600443.exe

**Comment by AnimatedMannequin** [comment]
Here are some builds you may use for testing: 

**self-hosted_debian-11_aarch64, full**
 - https://public.overte.org/build/overte/PR162/Overte-PR162-a6ac375.tar.gz
 - https://public.overte.org/build/overte/PR162/Overte-PR162-a6ac375.sh
 - https://public.overte.org/build/overte/PR162/Overte-PR162-a6ac375.tar.Z

**ubuntu-18.04, full**
 - https://public.overte.org/build/overte/PR162/Overte-PR162-a6ac375.sh
 - https://public.overte.org/build/overte/PR162/Overte-PR162-a6ac375.tar.Z
 - https://public.overte.org/build/overte/PR162/Overte-PR162-a6ac375.tar.gz

**macOS-10.15, full**
 - https://public.overte.org/build/overte/PR162/Overte-PR162-a6ac375.dmg

**windows-2019, full**
 - https://public.overte.org/build/overte/PR162/Overte-PR162-a6ac375.exe

File: cmake/ports/bullet3/portfile.cmake [hunk 1]
```diff
@@ -31,40 +29,12 @@ vcpkg_from_github(
...
-    endif()
-
-    set(VIRCADIA_BULLET_OPTIONS "${VIRCADIA_BULLET_OPTIONS}")
-
-    if(DEFINED VIRCADIA_CPU_ARCHITECTURE)
-        set(VIRCADIA_BULLET_OPTIONS "${VIRCADIA_BULLET_OPTIONS} -DCMAKE_CXX_FLAGS=\"${VIRCADIA_CPU_ARCHITECTURE}\" -DCMAKE_C_FLAGS=\"${VIRCADIA_CPU_ARCHITECTURE}\" ")
-    endif()
-endif()
-
-message("Optimization options for Bullet: ${VIRCADIA_BULLET_OPTIONS}")
```
Comment (ksuprynowicz) [inline cmake/ports/bullet3/portfile.cmake:58]: Won't this mean that Bullet will be again built entirely unoptimized on Linux and without architecture-specific flags?

File: cmake/ports/opus/portfile.cmake [hunk 2]
```diff
@@ -10,37 +8,13 @@ vcpkg_from_github(
...
-        set(VIRCADIA_OPUS_OPTIONS "-DCMAKE_BUILD_TYPE=RelWithDebInfo")
-    endif()
-
-    set(VIRCADIA_OPUS_OPTIONS "${VIRCADIA_OPUS_OPTIONS}")
-
-    if(DEFINED VIRCADIA_CPU_ARCHITECTURE)
-        set(VIRCADIA_OPUS_OPTIONS "${VIRCADIA_OPUS_OPTIONS} -DCMAKE_CXX_FLAGS=\"${VIRCADIA_CPU_ARCHITECTURE}\" -DCMAKE_C_FLAGS=\"${VIRCADIA_CPU_ARCHITECTURE}\" ")
-    endif()
-endif()
+  master
```
Comment (ksuprynowicz) [inline cmake/ports/opus/portfile.cmake:11]: Same thing as with Bullet.

**Review by ksuprynowicz — CHANGES_REQUESTED** [review]
I'm really worried about Bullet physics changes. Earlier by default neither Release nor RelWithDebInfo was set, making it extremely slow (up to 22 times from my tests). We will also need to test how this affects Opus performance.

**Comment by ksuprynowicz** [comment]
I will build it on Linux and test the perfromance soon.

File: cmake/ports/bullet3/portfile.cmake (hunk 1 above)
Comment (JulianGro) [reply cmake/ports/bullet3/portfile.cmake:58]: VCPKG sets those flags automatically, so there shouldn't be any difference.

File: cmake/ports/opus/portfile.cmake (hunk 2 above)
Comment (JulianGro) [reply cmake/ports/opus/portfile.cmake:11]: VCPKG sets those flags automatically, so there shouldn't be any difference.

File: cmake/ports/bullet3/portfile.cmake (hunk 1 above)
Comment (ksuprynowicz) [reply cmake/ports/bullet3/portfile.cmake:58]: There doesn't seem to be any difference on the Bullet performance. I will also test Opus soon.

File: cmake/ports/opus/portfile.cmake (hunk 2 above)
Comment (ksuprynowicz) [reply cmake/ports/opus/portfile.cmake:11]: Opus performance is fine too. I think this PR is safe to merge.

Comment (JulianGro) [reply cmake/ports/opus/portfile.cmake:11]: Give me an "Approved" and a "CR approved" then ;)

**Pull request merged**